tests/main.rs

```diff
@@ -250,3 +250,34 @@ async fn test_max_by_and_min_by() {
     - +---------------------+
     "###);
 }
+
+#[tokio::test]
+async fn test_kurtosis_pop() {
```

Comment:
It might be reasonable to add a few more tests to cover handling of null and empty values.

Reply:
My bad, i had add more.